controllers/Sitemap.js

```diff
@@ -0,0 +1,73 @@
+/* eslint-disable no-restricted-syntax */
+/* ------------------------------------------------------------------------------------------------
+ * Copyright (c) mgm security partners GmbH. All rights reserved.
+ * Licensed under the AGPLv3 License. See LICENSE.md in the project root for license information.
+ * ------------------------------------------------------------------------------------------------*/
+
+const xmlbuilder = require('xmlbuilder');
+const TagService = require('../service/TagService');
+const TestcaseService = require('../service/TestcaseService');
+
+module.exports.sitemapIndex = async (req, res) => {
+  try {
+    const PathDefault = 'https://canitrust.in/';
```

Comment:
Please change https://canitrust.in to https://www.canitrust.in
also for the tag path